public/js/components/ChartTheme/theme.js

```diff
@@ -0,0 +1,233 @@
+// *
```

Comment:
Maybe something in the README about some style living here as well as in the css?